experiment/depth-first-traversal/simulation/js/treePractice.js

```diff
@@ -57,7 +57,16 @@ function check(d){
     if(tree_traversal.sequence_list[index]){
   
     document.getElementById("commentbox").style.display = "block";
-    document.getElementById("comments").innerHTML = 'Wrong sequence.Try again! \n HINT: Next node in traversal: '+tree_traversal.sequence_list[index]
+    let traversaltype = document.getElementById("trav").value;
```

Comment:
Please use camelCase for variable names. This should be named as traversalType.
Please declare this as const and not let since we are not going to change this value/